packages/extension/.gitignore

```diff
@@ -1,26 +1,24 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-/node_modules
-/.pnp
-.pnp.js
-
-# testing
-/coverage
-
-# production
-/build
-
-# misc
-.DS_Store
-.env.local
```

Comment:
All the .env files need to be added back as we use them and we don't want them to be commuted with the secrets